apollo-router/src/query_planner/testdata/defer_clause.graphql

```diff
@@ -30,6 +30,8 @@ directive @link(
   import: [link__Import]
 ) repeatable on SCHEMA
 
+directive @defer(label: String, if: Boolean! = true) on FRAGMENT_SPREAD | INLINE_FRAGMENT
+
```

Comment:
```suggestion

```
defer should not be declared in the supergraph but added by API schema, if there's a test failure the test should use `Schema::parse_test`